docs/overview/global_ocean_circ.rst

```diff
@@ -0,0 +1,32 @@
+
+Global ocean circulation
+------------------------
+
+
+:numref:`large-scale-circ` shows the pattern of ocean
+currents at the surface of a 4° global ocean model run with
+15 vertical levels. Lopped cells are used to represent topography on a
+regular *lat-lon* grid extending from 70N to
+70S. The model is driven using monthly-mean winds with
+mixed boundary conditions on temperature and salinity at the surface.
+The transfer properties of ocean eddies, convection and mixing is
+parameterized in this model.
+
+  .. figure:: figs/isovec_atmoce_new.*
+    :width: 100%
+    :align: center
+    :alt: large-scale-circ
+    :name: large-scale-circ
+
+    Pattern of surface ocean currents from a global integration of the model at 4° horizontal resolution and with 15 vertical levels.
+
+:numref:`large-scale-circ2` shows the meridional overturning 
+circulation of the global ocean in Sverdrups.
+
+  .. figure:: figs/moc.*
```

Comment:
Splitting this figure in two means that the figure numbers following it don't match the current documentation.

If we are we trying to exactly reproduce the current docs, then this isn't ideal. However, I agree that two separate figures is a better way to present the results.

This comes up in a few other places too, but I haven't flagged them.